chapters/08-choose-n-patterns.md

```diff
@@ -198,10 +198,10 @@ analizeGold 585 = "Not bad! 585 standard."
 ```haskell
 analizeGold standard =
   case standard of
-    999       -> "Wow! 999 standard!"
-    750       -> "Great! 750 standard."
-    585       -> "Not bad! 585 standard."
-    otherwise -> "I don't know such a standard..."
+    999 -> "Wow! 999 standard!"
+    750 -> "Great! 750 standard."
+    585 -> "Not bad! 585 standard."
+    _   -> "I don't know such a standard..."
 ```
```

Comment:
Не просто упоминает. Переопределение — это почти ошибка.

```haskell
f x y = case x of
    True ->                 1
    otherwise ->
        if  | y == 2    ->  3
            | otherwise ->  4

main = print $ f False 1
```

Что выведет эта программа?